packages/shopping-cart/src/cart-functions.ts

```diff
@@ -1,5 +1,7 @@
+import { isDomainTransfer } from '@automattic/calypso-products';
```

Comment:
You can't use the `@automattic/calypso-products` package in the shopping cart package without adding it as a dependency, and I think that doing that here would be a bad idea. The shopping cart package is intentionally minimal and may be used for adding shopping cart functionality to wp-admin in the future, and the `calypso-products` package is massive. Is there another way to get the functionality you need here? Why are you even adding it to this package?